analytics/src/analytics/integrations/extracts/load_opportunity_data.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from io import BytesIO
+from urllib.parse import urlparse
+
+from analytics.integrations.etldb.etldb import etl_db_connection
+from analytics.integrations.extracts.constants import (
+    MAP_TABLES_TO_COLS,
+    OpportunityTables,
+)
+from analytics.integrations.extracts.s3_config import S3Config, get_s3_client
+
+logger = logging.getLogger(__name__)
+
+
+def get_s3_bucket(path):  # if not in env ?
+    return urlparse(path).hostname
```

Comment:
Is this used?

Reply:
Nope, not if we have it as env var. I will remove it.